### ISSUE TITLE
[NEW] Start app on Startup (Android)

### ISSUE THREAD

**Issue by djorkaeffalexandre** [RocketChat/Rocket.Chat.ReactNative#2121]
[Here](https://stackoverflow.com/questions/6391902/how-do-i-start-my-app-on-startup) have some explain about this, we need this as a setting that you can enable/disable.
It's just an Android feature because this isn't possible to do on iOS side.

**Comment by aptvaibhav** [comment]
Hi, i am new to this community can i work on this bug? And Can u guide me a little bit about this bug.

**Comment by githubbz** [comment]
Agreed, if you restart your phone you miss notifications, etc... Defn an important feature in my opinion.  Also for iOS your app auto starts automatically when VOIP tags are included. Now that RocketChat has voice call features might be worth looking into as an option to enable that feature and still be approved by Apple Dev review.

**Comment by aptvaibhav** [comment]
@djorkaeffalexandre can i work on this bug?

**Comment by djorkaeffalexandre** [comment]
Yes, go ahead.
You can find some details at the link.

**Comment by aptvaibhav** [comment]
okay

> Yes, go ahead.
> You can find some details at the link.

okay

**Comment by aptvaibhav** [comment]
can you please temme in which repository should i create this class(StartMyServiceAtBootReceiver) ? @djorkaeffalexandre
